data-explorer/kusto/management/workload-groups.md

```diff
@@ -52,6 +52,22 @@ You can't:
 
 You can [monitor](#monitoring) what gets classified to the `internal` workload group, and statistics of those requests.
 
+
+### Materialized-Views workload group
+
+The `materialized-views` workload group is populated with [materialized-views](materialized-views/materialized-view-overview.md) requests only.
```

Comment:
This is confusing since some might think it applies to queries over MVs as well. I would rephrase to something like: 
"The `$materialized views` workload group applies to materialized views materialization process (see how materialized views work)." - how materialized views work should be a link to the relevant section in the overview page